connector.py

```diff
@@ -16,6 +16,20 @@
 __all__ = ['ConnectorMatrix']
 
 
+def trim_reply_fallback_text(text: str) -> str:
```

Comment:
@tulir :smile:

Reply:
As far as I'm aware this is the first time the type hinting syntax has been introduced into the opsdroid code base. I feel like we should make a conscious decision to update the code base to use this or not, rather than introducing inconsistencies.

Reply:
I have pulled it for now.